### PR TITLE
Added a linter and checks

### PR DESCRIPTION
**Pull request by RunDevelopment**
This adds [stylelint](https://stylelint.io/) (a CSS linter) and a few checks in gulp as well as some minor changes.

__stylelint:__

Stylelint is the CSS linter I choose. It does what I want a CSS linter to do and that's it. If you know a better linter, we can switch anytime.

I configured the linter such that there are as few modifications as possible. The only things that changed in the CSS files are some whitespaces and newlines. All non-whitespace changes in the themes are errors that the linter pointed out.

To make the linter stricter, we just have re-enable the rules I disabled (all rules which are set to `null`). [Here](https://stylelint.io/user-guide/rules) is the documentation of all rules.

__gulp checks:__

I also added two checks, which 1) check that every theme has a screenshot and 2) that every theme (plus screenshot) is in the readme.

Both checks and the linter will be run by Travis. 
(Yay, Travis is actually doing something now!)

__Minor changes:__

* Fixed a typo in `package.json`.
* `npm ci`, as described in the README, doesn't work without `package-lock.json`, so I changed it to `npm i` and added the `package-lock.json` to the `.gitignore`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
   "name": "prism-themes",
   "version": "1.3.0",
-  "description": "Addtional themes for the Prism syntax highlighting library.",
+  "description": "Additional themes for the Prism syntax highlighting library.",
   "main": "README.md",
   "scripts": {
-    "test": "echo \"Error: no tests required\" && exit 0"
+    "lint": "npx stylelint \"themes/*.css\"",
+    "lint-fix": "npx stylelint \"themes/*.css\" --fix",
```
Comment (mAAdhaTTah) [inline package.json]: `npx` isn't required when running them from pkg.json scripts.

File: themes/prism-atom-dark.css [hunk 2]
```diff
@@ -104,7 +104,7 @@ pre[class*="language-"] {
 
 .token.entity {
 	color: #FFFFB6;
-	/* text-decoration: underline; */
+	cursor: help;
```
Comment (mAAdhaTTah) [inline themes/prism-atom-dark.css]: Why is this added?

File: themes/prism-atom-dark.css [hunk 3]
```diff
@@ -137,10 +137,7 @@ pre[class*="language-"] {
...
 	font-weight: bold;
 }
+
 .token.italic {
 	font-style: italic;
 }
-
-.token.entity {
-	cursor: help;
-}
```
Comment (mAAdhaTTah) [inline themes/prism-atom-dark.css]: Oh I see. Altho this still seems a bit odd but whatevs.

File: themes/prism-hopscotch.css [hunk 4]
```diff
@@ -9,8 +9,7 @@
 
 code[class*="language-"],
 pre[class*="language-"] {
-	color: #ffffff;
```
Comment (mAAdhaTTah) [inline themes/prism-hopscotch.css]: I'm assuming this is removed cuz it's the default color?

Comment (RunDevelopment) [reply themes/prism-hopscotch.css]: It's removed because at the end of rule is overwritten with `#b9b5b8`.

This is one of the "errors" (should I really call this an error? It's just non-functional code.) in the themes. Really hard to spot for humans but easy for linters.

File: package.json (hunk 1 above)
Comment (RunDevelopment) [reply package.json]: I didn't know that. Thanks for pointing it out!

**Comment by RunDevelopment** [comment]
🎉
